community-operators/elasticsearch-operator/elasticsearch-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,218 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: elasticsearch-operator.v0.0.1
+  namespace: placeholder
+  annotations:
+    categories: "OpenShift Optional, Logging & Tracing"
+    certified: "false"
+    description: |-
+      The Elasticsearch Operator for OKD provides a means for configuring and managing an Elasticsearch cluster for tracing and cluster logging.
+    containerImage: quay.io/openshift/origin-elasticsearch-operator:latest
+    createdAt: 2019-02-20T08:00:00Z
+    support: AOS Cluster Logging, Jaeger
+    alm-examples: |-
+        [
+            {
+                "apiVersion": "logging.openshift.io/v1alpha1",
+                "kind": "Elasticsearch",
+                "metadata": {
+                  "name": "elasticsearch"
+                },
+                "spec": {
+                  "managementState": "Managed",
+                  "nodeSpec": {
+                    "image": "quay.io/openshift/origin-logging-elasticsearch5:latest",
+                    "resources": {
+                      "limits": {
+                        "memory": "1Gi"
+                      },
+                      "requests": {
+                        "memory": "512Mi"
+                      }
+                    }
+                  },
+                  "nodes": [
+                    {
+                        "nodeCount": 1,
+                        "roles": ["client","data","master"], 
+                        "redundancyPolicy": "SingleRedundancy"
+                    }
+                  ]
+                }
+            }
+        ]
+spec:
+  displayName: Elasticsearch Operator
+
+  description: |
+    The Elasticsearch Operator for OKD provides a means for configuring and managing an Elasticsearch cluster for use in tracing and cluster logging.
+    This operator only supports OKD Cluster Logging and Jaeger.  It is tightly coupled to each and is not currently capable of
+    being used as a general purpose manager of Elasticsearch clusters running on OKD.
+
+    Once installed, the operator provides the following features:
+    * **Create/Destroy**: Deploy an Elasticsearch cluster to the same namespace in which the Elasticsearch custom resource is created.
+
```

Comment:
Are there any docs for using this operator with CLO or jaeger? Could these be included either in this description or the description of CLO/jaeger?

Reply:
I'm not sure what additional documentation applies other then:
```    The Elasticsearch Operator for OKD provides a means for configuring and managing an Elasticsearch cluster for use in tracing and cluster logging.
    This operator only supports OKD Cluster Logging and Jaeger.  It is tightly coupled to each and is not currently capable of
    being used as a general purpose manager of Elasticsearch clusters running on OKD.
```
What more is there to say?  The operator will act upon any `elasticsearch` that is created. It is immaterially that CLO or Jaeger created the resource.

Reply:
That verbiage in the description is sufficient